extensions/CocoStudio/Reader/SceneReader.h

```diff
@@ -62,8 +62,11 @@ class CC_EX_DLL SceneReader
 {
 
 public:
-	static SceneReader* sharedSceneReader();
-	static void purge();
+	CC_DEPRECATED_ATTRIBUTE static SceneReader* sharedSceneReader();
+	static SceneReader* getInstance();
+	CC_DEPRECATED_ATTRIBUTE static void purge();
+	static void destroyInstance();
+	static void destroyCocosStudio();
```

Comment:
Oh, there is a SceneReader::destroyInstance().
Why not add the logical of SceneReader::destroyCocosStudio() into SceneReader::destroyInstance()?